demo/components/demo/components/dropdowns/README.md

```diff
@@ -0,0 +1,9 @@
+Dropdowns offer items that the user can choose from.
+
+Add the `nglDropdown` directive to the `<div>` that wraps the dropdown list.
+
+Use the `nglDropdownTrigger` directive to any child element,
+usually a `<button>` or a `<a>`, to open or close the dropdown.
+
+Add the `nglDropdownItem` directive on any child element that constitutes
+a dropdown item.
```

Comment:
Can please document the accessibility and keyboard interactions? Take a look at the tabs readme if you want.